packages/twenty-server/src/modules/timeline/services/timeline-activity.service.ts

```diff
@@ -47,10 +51,16 @@ export class TimelineActivityService {
   }
 
   private async transformEvent(
-    event: ObjectRecordBaseEventWithNameAndWorkspaceId,
+    event: ObjectRecordBaseEvent,
+    workspaceId: string,
+    eventName: string,
   ): Promise<TransformedEvent[]> {
     if (['note', 'task'].includes(event.objectMetadata.nameSingular)) {
-      const linkedObjects = await this.handleLinkedObjects(event);
+      const linkedObjects = await this.handleLinkedObjects(
+        event,
```

Comment:
whenever we pass more than 1 argument to functions, I prefer to pass a single object as a parameter, I feel it greatly improves the DX (auto-complete vs having to check the function signature)

Here: transforEvent({ event, workspaceId, eventName}), same for handleLinkedObjects

Also, why do we need to pass the eventName independently?

Reply:
honestly, I don"t remember why I did that, good catch